Mathlib/Tactic/SimpRw.lean

```diff
@@ -54,9 +54,14 @@ by simp_rw [h1, h2]
 ```
 -/
 elab s:"simp_rw " cfg:(config)? rws:rwRuleSeq g:(location)? : tactic => do
-  -- TODO We should use the passed `cfg` here too,
-  -- but I don't know how to convert it back and forth from `Syntax`.
-  evalTactic (← `(tactic| simp%$s (config := { failIfUnchanged := false }) only $g ?))
+  let cfg' : TSyntax `Lean.Parser.Tactic.config ← (do
+    match cfg with
+    | Option.none =>
+      `(config| (config := ({ failIfUnchanged := false } : Lean.Meta.Simp.Config)))
+    | Option.some cfg =>
+      let cf : TSyntax `term := ⟨cfg.raw[3]⟩
```

Comment:
Would this work as something like
```suggestion
    | Option.some `((config := $cf)) =>
```

Reply:
I tried that also, indeed it would be much nicer, but I couldn't get it to work :/ if anyone knows how I'd love to learn!

Reply:
Ok I'll merge as is, if you find out how to make that syntax works please pr it and tag me :)

Reply:
I tried again, apparently the issue is resolved by not using a big match and instead just nesting the matches, ie.
#8592